docs/codeql/writing-codeql-queries/query-help-files.rst

```diff
@@ -165,6 +165,7 @@ Query help inclusion
 ====================
 
 To reuse content between different help topics, you can store shared content in one query help file and then include it in a number of other query help files using the ``include`` element. The shared content can be stored either in the same directory as the including files, or in ``SEMMLE_DIST/docs/include``.
+When a query help file is only included by other help files but does not belong to a specific query, it should have the file extension ``.inc.qhelp``.
```

Comment:
Do you think it makes sense to include this? Any feedback is appreciated.

Reply:
I'm not on the documentation team, but this looks good to me :+1: